app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
```

Comment:
Consider using parentheses for importing multiple classes to improve readability, as per the checklist: `Use ( ) while importing multiple classes/modules`.